### ISSUE TITLE
Implement new Catalog - Pass 1

### ISSUE THREAD

**Issue by astares** [pharo-spec/NewTools#228]

**Comment by MarcusDenker** [comment]
Does it make sense to keep this open?

**Comment by astares** [comment]
Yes - I have some code and work on it. Without an issue one can not use the "Create branch for issue" in Iceberg. That's why it already exists. 

You can tag it as "feature"
